pkg/cnab/provider/action_test.go

```diff
@@ -17,8 +18,9 @@ func TestAddReloccation(t *testing.T) {
 	require.NoError(t, err)
 
 	c := config.NewTestConfig(t)
-	instanceStorage := instancestorage.NewPluggableInstanceStorage(c.Config)
-	d := NewRuntime(c.Config, instanceStorage)
+	claimStorage := storage.NewTestClaimProvider()
+	credentialStorage := credentials.NewTestCredentialProvider(t, c)
+	d := NewRuntime(c.Config, claimStorage, credentialStorage)
```

Comment:
Ooh, let me look into that more.

Reply:
Can you take a look at the latest code that I just pushed, specifically NewTestRuntime? It got rid of all this boilerplate lines like the ones above. Now just NewTestRuntime and NewPorter have to deal with assembling claims and credential storage.

It all looks like this now:

https://github.com/carolynvs/porter/blob/a49e58c85764d8e0778538970600263e4a918456/pkg/cnab/provider/parameters_test.go#L54-L55

Now that I have that, I'm not sure I need/want a storage provider. Do you still need a need for it elsewhere in the code?

Reply:
Nice!  Looks great.

Agreed, at this stage it would probably be premature to consolidate the claims and creds providers into one storage provider.  Can't yet come up with a need other than consolidation/reduction of function arguments at this time.